### PR TITLE
Minor fixes (proposals)

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: plugins/importexport/pubmed/locale/es_ES/locale.xml [hunk 1]
```diff
@@ -12,22 +12,22 @@
...
 	<message key="plugins.importexport.pubmed.description">Exportación de la metadata de artículos usando XML de PubMed para Indexación en MEDLINE.</message>
 	
-	<message key="plugins.importexport.pubmed.export">Exportar Datos</message>
-	<message key="plugins.importexport.pubmed.export.issues">Exportar Números</message>
-	<message key="plugins.importexport.pubmed.export.selectIssue">Seleccione un Número para exportar.</message>
-	<message key="plugins.importexport.pubmed.export.articles">Exportar Artículos</message>
-	<message key="plugins.importexport.pubmed.export.selectArticle">Seleccione Artículos para exportar.</message>
+	<message key="plugins.importexport.pubmed.export">Exportar datos</message>
+	<message key="plugins.importexport.pubmed.export.issues">Exportar números</message>
+	<message key="plugins.importexport.pubmed.export.selectIssue">Seleccione unnNúmero para exportar.</message>
```
Comment (asmecher) [inline plugins/importexport/pubmed/locale/es_ES/locale.xml]: I think there is a typo here!

Comment (kant) [reply plugins/importexport/pubmed/locale/es_ES/locale.xml]: `unnNúmero` should be `un Número`

Comment (asmecher) [reply plugins/importexport/pubmed/locale/es_ES/locale.xml]: That's what I thought, thanks -- could you update the PR?

Comment (kant) [reply plugins/importexport/pubmed/locale/es_ES/locale.xml]: Done @asmecher

Comment (asmecher) [reply plugins/importexport/pubmed/locale/es_ES/locale.xml]: Thanks, @kant!

**Comment by asmecher** [comment]
Merged and cherry-picked to `stable-3_1_2`, thanks!

**Pull request merged**